tests/unit/test.program.js

```diff
@@ -911,7 +911,15 @@ describe('program.defaultVersionGetter', () => {
   });
 
   it('returns git commit information in development', function() {
-    const commit = `${git.branch()}-${git.long()}`;
+    var commit;
```

Comment:
could we check if the .git directory does actually exist (at the start of this test case) and skip it if the .git directory doesn't exist instead?

Reply:
Thanks for the suggestion. It sounds better and I use that approach in the latest revision of this patch.